### PR TITLE
Added functionality to be able to send Interactive Whatsapp messages.

### PR DESCRIPTION
**Pull request by Alkiimista**
Now able to create Interactive Whatsapp messages.

### CODE REVIEW

File: CMText/version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #project Version
-__version__ = '1.1.3'
\ No newline at end of file
+__version__ = '1.2.1'
```
Comment (EnessenE) [inline CMText/version.py]: Why 1.2.1? Why not 1.1.4 or 1.2.0 or did I miss a version

File: README.md [hunk 2]
```diff
@@ -70,6 +70,9 @@ By calling `AddWhatsappTemplateMessage` and providing `Template`, sender name, r
 
 See Examples folder for more examples.
 
+## Sending Interactive Whatsapp message
+See the examples folder for an example.
```
Comment (EnessenE) [inline README.md]: (arguebaly you an add a really simple example here or a direct link to the folder but thats just me)

**Review by EnessenE — COMMENTED** [review]
Left a question and a comment

File: CMText/version.py (hunk 1 above)
Comment (Alkiimista) [reply CMText/version.py]: Changed the version number to 1.2.0

File: README.md (hunk 2 above)
Comment (Alkiimista) [reply README.md]: Added a direct link to the file.

**Pull request merged**